docs/developer/process/integration-release.md

```diff
@@ -60,38 +62,40 @@ To see all checks that need to be released, run `ddev release show ready`.
 
     This will automatically:
 
-    - update the version in `<INTEGRATION>/datadog_checks/<INTEGRATION>/__about__.py`
-    - update the changelog
-    - update the `requirements-agent-release.txt` file
-    - update [in-toto metadata](../meta/cd.md)
-    - commit the above changes
+    * update the version in `<INTEGRATION>/datadog_checks/<INTEGRATION>/__about__.py`
+    * update the changelog
+    * update the `requirements-agent-release.txt` file
+    * update [in-toto metadata](../meta/cd.md)
+    * commit the above changes
 
-1. Push your branch to GitHub and create a pull request.
+    ### Third party
 
-    1. Update the title of the PR to something like `[Release] Bumped <INTEGRATION> version to <VERSION>`.
-    1. Ask for a review in Slack.
+    * Update the version on `datadog_checks/<INTEGRATION>/__about__.py`.
+    
+    * Update the CHANGELOG.md file This file can be automatically updated by `ddev` using the following command:
+    
+    ```
+    ddev release changelog <INTEGRATION_NAME> <VERSION>
+    ``` 
 
-1. Merge the pull request after approval.
+    This command will list all merged PRs since the last release and creates a changelog entry based on the pull request labels,
+    this means that *the version bump needs to be on a separate PR from the one that included the changes*.
+    For changelog types, we adhere to those defined by [Keep a Changelog](https://keepachangelog.com/en/1.1.0/#how).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
 
-### PyPI
 
-If you released `datadog_checks_base` or `datadog_checks_dev` then these must be uploaded to [PyPI][]
-for use by [integrations-extras][].
+3. Push your branch to GitHub and create a pull request.
 
-This is automatically handled by two GitHub Action jobs: [release-base.yml][] and [release-dev.yml][].
-
-In case you need to do it manually:
+    1. Update the title of the PR to something like `[Release] Bumped <INTEGRATION> version to <VERSION>`.
+    1. Ask for a review in Slack.
 
-```
-ddev release upload datadog_checks_[base|dev]
-```
+4. Merge the pull request after approval or wait for it to be merged.
 
-### Metadata
+## Metadata
 
-You need to run certain jobs if any changes modified integration metadata. See the
-[Declarative Integration Pipeline](https://github.com/DataDog/devops/wiki/Declarative-Integration-Pipeline) wiki.
+You need to run certain backend jobs if any changes modified integration metadata or assets such as dashboards.
+If you are a contributor a datadog employee will handle this.
```

Comment:
```suggestion
You need to run certain backend jobs if any changes modified integration metadata or assets such as dashboards. See the[Declarative Integration Pipeline](https://github.com/DataDog/devops/wiki/Declarative-Integration-Pipeline) wiki.

!!! note
    If you are a contributor a datadog employee will handle this.
```

Should we keep the link somewhere for Datadog employees to sync themselves?

Reply:
I'm not sure because this is public accessible and the other is not.

Reply:
Make sense, but then we should make sure they know what to do to sync the metadata by reading our team internal wiki